macros/sql/get_column_values.sql

```diff
@@ -1,8 +1,8 @@
-{% macro get_column_values(table, column, order_by='count(*) desc', max_records=none, default=none) -%}
-    {{ return(adapter.dispatch('get_column_values', 'dbt_utils')(table, column, order_by, max_records, default)) }}
+{% macro get_column_values(table, column, where=none, order_by='count(*) desc', max_records=none, default=none) -%}
+    {{ return(adapter.dispatch('get_column_values', 'dbt_utils')(table, column, where, order_by, max_records, default)) }}
 {% endmacro %}
 
-{% macro default__get_column_values(table, column, order_by='count(*) desc', max_records=none, default=none) -%}
+{% macro default__get_column_values(table, column, where, order_by='count(*) desc', max_records=none, default=none) -%}
```

Comment:
I'd prefer to add this as the last optional argument to avoid a breaking change for any consumers using positional arguments.

Consumers using keyword arguments can still supply them in any order.

This suggestion pairs with the previous one:

```suggestion
{% macro default__get_column_values(table, column, order_by='count(*) desc', max_records=none, default=none, where=none) -%}
```